### PR TITLE
Add support for managing aptly mirrors

### PR DESCRIPTION
**Pull request by xaoc7**

### CODE REVIEW

File: aptly_api/parts/snapshots.py [hunk 1]
```diff
@@ -49,7 +49,16 @@ def create_from_repo(self, reponame: str, snapshotname: str, description: Option
         if description is not None:
             body["Description"] = description
 
-        resp = self.do_post("api/repos/%s/snapshots" % quote(reponame), json=body)
+        resp = self.do_post("api/repos/%s/snapshots" %
+                            quote(reponame), json=body)
```
Comment (jdelic) [inline aptly_api/parts/snapshots.py:53]: These newlines don't really fit with the style across the other files... is this just your style or something like `black`?

Comment (xaoc7) [reply aptly_api/parts/snapshots.py:53]: They come from my IDE (auto formatted). I'm using Atom, unfortunately with default line lengths etc..I can fix that if needed.

**Comment by xaoc7** [comment]
@jdelic Any chance to merge this soon?

**Comment by xaoc7** [comment]
@jdelic I'm sorry to bothering you,but it seems that you have no time to maintain that product anymore? Any chance to bring it back to live?

**Comment by GregoryLand** [comment]
I would also like to see this merged and a new release made with this patch set.

**Comment by cfiehe** [comment]
Also a 👍 from my side for merging this pull request. It would be great enhancement.

**Comment by GregoryLand** [comment]
@xaoc7 While doing some work in my own branched I noticed an issue with this patch set.  parts/mirrors.py: mirrors.create argument filter_with_deps is not connected to anything. To me it looks like it should be connected to data[filter_with_deps].

**Comment by xaoc7** [comment]
> @xaoc7 While doing some work in my own fork I noticed an issue with this patch set. parts/mirrors.py: mirrors.create argument filter_with_deps is not connected to anything. To me it looks like it should be connected to data["FilterWithDeps"].
> 
> It also looks like parts/snapshots.py create_from_mirror description is also not connected.

I agree, but I'm not sure if it is worthy to fix those since my PR is more than an year old and it seems that there is no willing to be merged. In other words I'm not seeing a reason to contribute to a dead project.

**Comment by cfiehe** [comment]
@jdelic Are there any chances to bring this project back to an active state and get the pull requests reviewed and merged?

**Comment by GregoryLand** [comment]
@xaoc7 to follow up on my previous comment I have sent a pull request to your master branch.  I have corrected the issues I found and fixed the tests. https://github.com/xaoc7/aptly-api-client/pull/1

**Comment by jdelic** [comment]
This PR has been merged as #126 as it required some formatting fixes and subsequent modifications from repo forks. Thank you all so much for supporting this.

**Pull request closed without merging**